man/tpm2_print.1.md

```diff
@@ -0,0 +1,51 @@
+% tpm2_print(1) tpm2-tools | General Commands Manual
+%
+% DECEMBER 2018
```

Comment:
I don't think this patch will take a year to be merged :smiley:

Reply:
:) fixed